### PR TITLE
Fix IPC::Run perl module name in CI

### PR DESCRIPTION
**Pull request by artemgavrilov**
```
Warning: Cannot install IPC::RUN, don't know what it is.
Try the command

    i /IPC::RUN/

to find objects with matching identifiers.
```

https://metacpan.org/pod/IPC::Run

Same for pg_tde: https://github.com/Percona-Lab/pg_tde/pull/156

### CODE REVIEW

File: .github/workflows/postgresql-12-pgdg-package.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
             /usr/lib/postgresql /usr/include/postgresql /usr/share/postgresql \
             /etc/postgresql
           sudo rm -f /usr/bin/pg_config
-          sudo /usr/bin/perl -MCPAN -e 'install IPC::RUN'
+          sudo /usr/bin/perl -MCPAN -e 'install IPC::Run'
```
Comment (artemgavrilov) [inline .github/workflows/postgresql-12-pgdg-package.yml:26]: We installing perl modules in `Delete old postgresql files` step, that looks wrong. Should we extract these actions to separate step?

**Comment by artemgavrilov** [comment]
CI jobs that are failed it's same set that fails on main branch.

**Review by Naeem-Akhter — APPROVED** [review]
Looks Good!

File: .github/workflows/postgresql-12-pgdg-package.yml (hunk 1 above)
Comment (Naeem-Akhter) [reply .github/workflows/postgresql-12-pgdg-package.yml:26]: No harm in doing it. But as these are sequential steps so either way we are likely to see same behaviour.

**Review by dAdAbird — APPROVED** [review]

**Pull request merged**